2024 governance update (#7023)
OpenRefine/OpenRefine: GOVERNANCE.md

# OpenRefine Governance Model

## Summary / Overview
OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a [plugin architecture](https://openrefine.org/docs/technical-reference/writing-extensions) and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.

## Roles and responsibilities
OpenRefine is a meritocratic, consensus-based community project based on the [Apache Decision Making process](https://community.apache.org/committers/decisionMaking.html).

> In so far that is possilbe, decision-making is done based on user consensus following open discussion (i.e., using OpenRefine's [discussion list](https://forum.openrefine.org/) and [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)). No major decisions about the project’s direction, bug fixes, or features should be made in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.

Reviewer: ```suggestion
In so far that is possible, decision-making is done based on user consensus following open discussion (i.e., using OpenRefine's [discussion forum](https://forum.openrefine.org/) and [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)). No major decisions about the project’s direction, bug fixes, or features should be made in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.
```

All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).

### Users
Users are community members who have a need for the project. Through their usage, they give the project a purpose. Users are encouraged to participate in the project life by providing feedback on how their needs are satisfied.

Users can help the project by:

- Advertising and advocating for the project
- Informing developers of the strengths and weaknesses of the tool through the [user discussion list](https://forum.openrefine.org/) or the [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)
- Providing moral support (a ‘thank you’ goes a long way)
- Writing tutorials to contextualize OpenRefine for specific communities

How to become an OpenRefine user? [Download OpenRefine](http://openrefine.org/download.html) and start refining!

### Contributors
Contributors are users getting involved in the project more closely. Contributions can take many forms:

- Supporting new users via the [user discussion list](https://forum.openrefine.org/c/support/12)

> - Submitting patches to fix bugs or add features via pull requests, comment on pull requests and help other contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8)

Reviewer: The previous bullet about peer support seems like it belongs in the Users section, rather than Contributors. Also, a couple of grammatical fixes here:

```suggestion
- Submitting patches to fix bugs or add features via pull requests, commenting on pull requests and helping other contributors via the [developer discussion list](https://forum.openrefine.org/c/dev/8)
```

- Maintaining and improving the [website](https://openrefine.org/)
- Writing and maintaining the [documentation](https://github.com/OpenRefine/openrefine.org/tree/master/docs)

> - Helping [design interfaces](https://openrefine.org/docs/technical-reference/introduction) for OpenRefine

Reviewer: ```suggestion
- Helping [design user interfaces](https://openrefine.org/docs/technical-reference/introduction) for OpenRefine
```

I'm guessing this is user interfaces as opposed to programming interfaces, but clarify either way.

How to become an OpenRefine contributor? You will find more details in our [contributing guideline](https://github.com/OpenRefine/OpenRefine/blob/master/CONTRIBUTING.md)

### Committers
If you make regular contributions to OpenRefine, you will most likely become a Committer.

Committers have earned enough trust from the community to

- Triage issues, pull requests and [projects](https://github.com/OpenRefine/OpenRefine/projects)
- Have direct access to the code base to create new branches

> - Organize the GitHub wiki

Reviewer: This seems like it could be split up. Editing the wiki (which I thought had been retired) and triaging issues are less technically demanding tasks that don't need to be done by committers. Similarly, OpenRefine org membership seems like it could be decoupled from the type of contributor (although it may require adjustment to the privileges associated with it).

Author: The GitHub wiki still has several relevant documents (I did an audit when [preparing the contributor pathways](https://forum.openrefine.org/t/openrefine-2024-barcamp-improving-contributor-pathways/1630) and [listed them here](https://docs.google.com/document/d/1HIYA0fqOabjlxVPwdhqTpJD_0-TMs30Nk38pTGqJunE/edit?tab=t.0#heading=h.r3vv2g7bxk20)). I have been trying to migrate them to the documentation when possible. It is a lower priority and is not moving as fast as I wish. Due to the migration process, I think it is best to limit the wiki organization to committers (even if currently anyone with a GitHub account can edit). 

Currently, to add a label to issues, you need to be part of the GitHub organization (so be a committer).

- Review pull requests submitted by contributors
- Run the CI pipeline on your PRs automatically
- Are part of the OpenRefine organization and have the OpenRefine badge on their GitHub profile

> **How to become a Committer?**: Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.

Reviewer: ```suggestion
**How to become a Committer?**: Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer section of the forum](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.
```

> **Current list of Committers**: The list is available here: https://github.com/orgs/OpenRefine/people.

Reviewer: That list needs serious pruning. 77 committers is an insanely high number given the number of people active on the project (and how long it's been since some of those people have been active).

### Core Developer Group
Also referred to as Core Dev. The Core Developer Group are trusted developers who
- Merge pull requests
- Publish releases
- Review security vulnerability reports
- Provide long-term project vision with the community
- Are responsible for the technical direction of the project
- Are part of the Admin team for the project on GitHub

> **How to join the Core Dev Group?**. To be eligible to serve on the Core Developers, individuals must first be Committers of the project. Core Dev Group members are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.

Reviewer 2: ```suggestion
**How to join the Core Dev Group?**. To be eligible to serve on the Core Developers, individuals must first be Committers of the project. Core Dev Group members are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where committers can nominate themselves or other committers. See the Member Election section below for more details.
```
Small change since the hierarchy is that only committers can become part of core dev, and not contributors directly who must first be committers then part of ccontributors. Change enforces that there cannot be any direct promotions into core dev by contributers.

Reviewer 3: Good catch! That makes it more clear indeed.

Author: Thanks @thadguidry

**Current list of Core Developers:**
- Tom Morris [@tfmorris](https://github.com/tfmorris) and on the [forum](https://forum.openrefine.org/u/tfmorris/summary)

> - Antonin Delpeuch until March 2025: [@wetneb](https://github.com/wetneb) and on the [forum](https://forum.openrefine.org/u/antonin_d/summary).

Reviewer: The date for Antonin has already passed. Is it still valid? 

We need to figure out a way to bootstrap this group. One, or even two, members is too few. @Abbe98 his provided useful technical feedback in the past. Perhaps he could be a candidate, if willing.

#### Release Manager
The Release Manager is responsible to
* Coordinate with the community to select which issues and pull requests are part of new releases
* Prepare and coordinate publishing new releases

The current Release Manager is [Antonin Delpeuch](https://github.com/wetneb).

### Code of Conduct Committee
The Code of Conduct Committee is responsible for responding to all [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md) reports, as well as managing and maintaining the Code of Conduct policy document.

**How to join the Code of Conduct Committee?**. Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.

### Advisory Committee

> The Advisory Committee runs the administrative aspect of the project on a daily basis with the support of Code for Science and Society (CS&S). Its members are bound by the fiscal sponsorship agreement with Code for Science and Society. They meet once per month with Code for Science and Society.

Reviewer: ```suggestion
The Advisory Committee runs the administrative aspect of the project on a regular basis with the support of Code for Science and Society (CS&S). Its members are bound by the fiscal sponsorship agreement with Code for Science and Society. They meet once per month with Code for Science and Society.
```

I would suggest `regular` or `ongoing` or some other term which doesn't imply daily actiivty.

**How to join the Advisory Committee?** Members of the Advisory Committee publish an announcement on the blog when searching for a new member. The Advisory Committee selects new committee members through consensus-driven discussions among current members.

**Current list of Advisory Committee members**
- Jan Ainali [@Ainali](https://github.com/Ainali) and on the [forum](https://forum.openrefine.org/u/ainali/summary)
- Julie Faure-Lacroix [@jfaurelacroix](https://github.com/jfaurelacroix) and on the [forum](https://forum.openrefine.org/u/jfaurel/summary)
- Esther Jackson [@ej2432](https://github.com/ej2432) and on the [forum](https://forum.openrefine.org/u/ej2432/summary)

### Project Manager

> OpenRefine's Project Manager (paid position) works closely with OpenRefine's advisory committee, Core Dev Groups and OpenRefine users and contributors.

Reviewer: ```suggestion
OpenRefine's Project Manager (paid position) works closely with OpenRefine's Advisory Committee, Core Dev Group, and OpenRefine users and contributors.
```

The Project Manager:

> - Helps to improve OpenRefine's governance and community diversity

Reviewer: ```suggestion
- Helps improve OpenRefine's governance and community diversity
```

  - CS&S requires the committee to be at least three people.
  - Every member of the Advisory Committee must sign the fiscal sponsorship agreement with CS&S, agreement, and agree to the policies of CS&S.
  - Must supervise (manage pay and review performance) paid project staff or delegate to the Project Manager.
- Manage Budget & Contract
  - Approve budget and oversee grant and financial activities
  - Approve grant applications
  - Enter into contracts on behalf of the project
  - Oversee contract and legal engagement for the project
- Participate in at least one call per quarter;
- Publish minutes of their meeting on [the forum](https://forum.openrefine.org/tag/minute-advisory)
- Advisory Committee members are subject to the Conflict of Interest policy.

## Fiscal sponsorship: Code For Science and Society
Since January 2020, OpenRefine is a member project of [Code For Science and Society](https://codeforscience.org/) (CS&S). CS&S provides administrative, legal and fiscal infrastructure to receive and manage funds under the CS&S Fiscal Sponsorship Agreement.
- CS&S retains a 15% handling fee on any donation to finance their operations (20% for government funding).

> - CS&S owns and manages the domain openrefine.org

Reviewer: Hopefully/presumably as a trustee for the OpenRefine project.

**Current list of Advisory Committee members**
- Jan Ainali [@Ainali](https://github.com/Ainali) and on the [forum](https://forum.openrefine.org/u/ainali/summary)
- Julie Faure-Lacroix [@jfaurelacroix](https://github.com/jfaurelacroix) and on the [forum](https://forum.openrefine.org/u/jfaurel/summary)
- Esther Jackson [@ej2432](https://github.com/ej2432) and on the [forum](https://forum.openrefine.org/u/ej2432/summary)

### Project Manager
OpenRefine's Project Manager (paid position) works closely with OpenRefine's advisory committee, Core Dev Groups and OpenRefine users and contributors.

The Project Manager:
- Helps to improve OpenRefine's governance and community diversity
- Helps build community-driven structures to formalize OpenRefine's roadmap and keep it up-to-date
- Assists in finding and securing new sources of funding
- Supports the Advisory Committee in OpenRefine's day-to-day operations; this may include supervising paid project staff.

The current Project Manager is Martin Magdinier [@magdmartin](https://github.com/magdmartin) and on the [forum](https://forum.openrefine.org/u/Martin/summary).

> # OpenRefine Policies

Reviewer 2: I kind of strongly feel this should be renamed as
```suggestion
# Advisory Committee Policies
```

Author: Some of those policies also apply to the CoC and potentially to the Core Dev Group. 
If we want something more neutral we can just name the section `Policies`

Author: I merged this suggestion by mistake. I am reverting it in with my latest commit.

> Introduction to Governance Policies

Author: ```suggestion
Introduction to the Policies
```

> Introduction to Governance Policies

Author: Some of the policies mentioned below also apply to the CoC, Core Dev Group, and Committer regarding the voting policies. This is explained in the following section. 

However, I understand your concern regarding the "Governance Policies" terms and suggest replacing it by just `Policies`

> Introduction to Governance Policies

Author: I managed to incorporate @thadguidry  suggestion in the document.

The **Code of Conduct Committee, Advisory Committee, and Core Developer Group** hold greater authority within the OpenRefine project, with the ability to represent and make commitments on its behalf. To maintain the community’s trust, these groups are held to a higher standard of transparency and accountability in their decision-making processes.
This includes clear and open practices for:
- Member Election 
- Requesting and managing funds
- Managing conflicts of interest

> ## Member Election

Reviewer 2: This **Member Election** concerns itself with Committers, Core Developer Group, and Code of Conduct Committee members which is part of OpenRefine Governance (not Polices), and henceforth should be moved to after the **Release Manager** section to detail how the Governance groups are elected.

Advisory Committee elections and policies can remain in the **OpenRefine Policies** section.
Heck, I'd even go so far as to NOT call the section **OpenRefine Policies** but instead **Advisory Committee Policies** please.

Author: We agree here. I think the conversation is mostly about how we structure the document. I am ok with either:
1) Having the voting part in the core governance document (and removing mention of committers and Core Dev Group from the policy section)
2) Keeping things as is, such as mentioning the Committers group on line 103. 

I'll be interested in having other opinions on how to structure this best; @Ainali, maybe?

Author: I am keeping things as is to avoid more edits in the document. Again, I am happy to revisit my position.

Reviewer 3: It could stay here, but since the elections touch upon the groups we mention higher up, it makes sense to me to move this section up there.

Author: This was discussed during t[the April 10 AC](https://forum.openrefine.org/t/april-10-2024-advisory-committee/2230), and the consensus was to move it to the governance section, which is done with my latest commit.

# OpenRefine Governance Model

## Summary / Overview
OpenRefine is a free, open-source, powerful tool for working with messy data. OpenRefine has a [plugin architecture](https://openrefine.org/docs/technical-reference/writing-extensions) and is distributed under the [new BSD license](http://opensource.org/licenses/BSD-3-Clause) allowing modification, distribution and name changes.

## Roles and responsibilities
OpenRefine is a meritocratic, consensus-based community project based on the [Apache Decision Making process](https://community.apache.org/committers/decisionMaking.html).

> Decision-making is done based on user consensus following open discussion (i.e., using OpenRefine's [discussion list](https://forum.openrefine.org/c/dev/8) and [issue list](https://github.com/OpenRefine/OpenRefine/issues?state=open)). No decisions about the project’s direction, bug fixes, or features may be made in private without community involvement and participation. Discussions must begin at the earliest possible point on a topic; the community’s participation is vital during the entire decision-making process.

Reviewer 4: > No decisions about the project’s direction, bug fixes, or features may be made in private without community involvement and participation.

This is not really align with our practice or the Apache Decision Making process. There are a lot of reasons to decide certain things in private. Security, actual consensus making, etc. If we want to keep it simple lets just link to the Apache document without trying to patch it?

Author: I imported the wording from our initial governance as I wanted to minimize the diff. While I still hope to get this merge sooner than later, I suggest making more substantial edits in a separate PR.

> All project participants abide by the [Code of Conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md).

Reviewer 4: Is the CoC up to date? Last time (which I know of) it was "enforced" by an AC member not listed in that doc...

Reviewer 5: I think the members could be refreshed (for instance, as Jessica has left CS&S), but to me it still reflects the current state of the committee. Are you worried that the CoC committee could be just a sock puppet of the AC without actual independence and agency? If you are referring to me as a former AC member, last time I reached out to the CoC about a particular situation, they disagreed about my assessment of that situation and followed their own judgment, which I would take as a good sign of the committee's independence.

That being said, it's an open question to me how to keep a CoC committee active for such a small project because it only gets woken up once in a blue moon, making it hard to keep the practice and understandings of the tasks alive. Surely there is room for improvement, but that's something we should rather do in another PR I would say.

Reviewer 3: Whether or not the CoC is mentioned here or not doesn't change its status I would hope. This is just a link for convenience.

Reviewer 4: @Ainali the CoC contains quite a lot about governance and it's currently very badly aligned with this document, who are considered a "project leader"? can the CoC group enforce rights-changes against contributors? It's all governance, and with the ACs history of weaponizing it, I would ague it's rather important to get it right.

Reviewer 6: @Abbe98 : “with the ACs history of weaponizing it”
What do you mean? That the CoC has been use by the AC?

“project leader”: do you mean project leadership? Because that seams to me to be appropriate even if not defined.

The CoC is pretty general and more guidelines than a checklist/every step procedure. It did suits us, the project, in the last 4 years that I've participated in the committee.

Regards, Antoine

Author: @antoine2711  I think @Abbe98 refer to this sentence
> Project contributors who do not follow or enforce the Code of Conduct in good
faith may face temporary or permanent repercussions as determined by other
members of the project's leadership.

I agree that it can be more explicit who who can make the decision (CoC member only), or we can just remove it entirely.  

@antoine2711  I would also welcome update a yearly report frequency from the CoC (number of report received, update to the policy and change of membership) so its operation is transparent to the community.

> ### Users

Reviewer 4: These sections create expectations around people and try to sort them into buckets of with little to no relevance to the governance. I would suggest limiting these sections to match those of GitHub contributors, it would simplify this a lot while staying true to the actual state of order.

Reviewer 3: Your comment implies that this governance file should be limited in scope to just the GitHub organization. That may be one way to do it, but then I think we need another governance file covering the entire project/community and not just the parts happening on GitHub. While each of the files may be simplified, it will be harder to get an overview both of the actual state and the intended state.

Reviewer 4: @Ainali you don't need to sort people and their work into buckets to have a governance document, removing these sections wouldn't change the policy at all.

I do argue for the governance document to be split as the current approach makes things unclear and the later sections contradicts the former ones.

Author: I imported the wording from our initial governance as I wanted to minimize the diff. While I still hope to get this merge sooner than later, I suggest making more substantial edits in a separate PR.

### Committers
If you make regular contributions to OpenRefine, you will most likely become a Committer.

Committers have earned enough trust from the community to

- Triage issues, pull requests and [projects](https://github.com/OpenRefine/OpenRefine/projects)
- Have direct access to the code base to create new branches
- Organize the GitHub wiki
- Review pull requests submitted by contributors
- Run the CI pipeline on your PRs automatically
- Are part of the OpenRefine organization and have the OpenRefine badge on their GitHub profile

**How to become a Committer?**: Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the Member Election section below for more details.

**Current list of Committers**: The list is available here: https://github.com/orgs/OpenRefine/people.

### Core Developer Group
Also referred to as Core Dev. The Core Developer Group are trusted developers who
- Merge pull requests
- Publish releases
- Review security vulnerability reports

> - Provide long-term project vision with the Advisory Committee team

Author: ```suggestion
- Provide long-term project vision with the community
```

This is done via public discussions on the forum, on github, using various user survey (see the [feature survey 2024](https://forum.openrefine.org/t/results-from-the-feature-prioritization-survey-2024/1847) for example) and roadmap exercise (see [barcamp notes](https://forum.openrefine.org/t/openrefine-2024-barcamp-strategizing-openrefine-roadmap/1633)) or [proposed roadmap](https://forum.openrefine.org/t/prioritisation-of-wikimedia-sveriges-development-work/1803)

#### Release Manager
The Release Manager is responsible to
* Coordinate with the community to select which issues and pull requests are part of new releases
* Prepare and coordinate publishing new releases

The current Release Manager is Tom Morris [@tfmorris](https://github.com/tfmorris) and on the [forum](https://forum.openrefine.org/u/tfmorris/summary).

### Code of Conduct Committee
The code of conduct committee is responsible to
- Respond to [code of conduct](https://github.com/OpenRefine/OpenRefine/blob/master/CODE_OF_CONDUCT.md) report
- Provide at least a quarterly report on the forum [day-to-day operations](https://forum.openrefine.org/c/community/operations/10)

**How to join the Code of Conduct Committee?**. Committers are voting-binding members; they select and elect new members through a voting process. Nominations should be submitted to the [developer discussion list](https://forum.openrefine.org/c/dev/8), where contributors can nominate themselves or other contributors. See the New Member Election section below for more details.

> ### Advisory Committee

Reviewer 4: From here on I think the document do no longer describe the governance of the project but instead the relationship between CS&S and the AC. This is clearly not aligned with the Apache Decision Making process nor do it concern the open source project.

I would suggest moving this somewhere else.

Reviewer 5: To me the AC is currently a component of the project, so it makes sense that it's described in this document. I think @magdmartin's intention is to first update the governance document to reflect the reality of the current situation, and currently the AC does feel like a part of the project to me, so I would keep it. The intention is not to set this in stone, but do incremental updates as the subject is contentious.

There could be further changes later on: changing the way the AC works, or deleting it entirely and leaving CS&S to go back to being just a GitHub project. I think it's best to discuss those separately.

Reviewer 4: > and currently the AC does feel like a part of the project to me, so I would keep it

This is probably because you are a part of it? You are the only notable GitHub contributor who are also a part of the AC...

Author: Following discussion with @thadguidry  we moved AC specific information to the Policy section. We agreed to store Policies in the same document to make them easily accessible  to the community and as we lack structure to host them (it may be a new section in the documentation, dedicated repository, Google Drive space).

**Current list of Advisory Committee members**
- Jan Ainali [@Ainali](https://github.com/Ainali) and on the [forum](https://forum.openrefine.org/u/ainali/summary)
- Julie Faure-Lacroix [@jfaurelacroix](https://github.com/jfaurelacroix) and on the [forum](https://forum.openrefine.org/u/jfaurel/summary)
- Esther Jackson [@ej2432](https://github.com/ej2432) and on the [forum](https://forum.openrefine.org/u/ej2432/summary)

### Project Manager
OpenRefine's Project Manager (paid position) works closely with OpenRefine's advisory committee, Core Dev Groups and OpenRefine users and contributors.

The Project Manager:
- Helps to improve OpenRefine's governance and community diversity
- Helps build community-driven structures to formalize OpenRefine's roadmap and keep it up-to-date
- Assists in finding and securing new sources of funding
- Supports the Advisory Committee in OpenRefine's day-to-day operations; this may include supervising paid project staff.

The current Project Manager is Martin Magdinier [@magdmartin](https://github.com/magdmartin)and on the [forum](https://forum.openrefine.org/u/Martin/summary).

> ## Member Election

Reviewer 3: Is this the New Member Election section referred to above in several places? If so, it should probably be named that.

Author: @Ainali, thanks for spotting this. I adjusted how we reference the section through the document

When voting for new Committers, Core Developer Group or Code of Conduct Committee members, start a new thread with a subject line starting with `[VOTE]` to indicate that a formal vote has been requested on the [developer discussion list](https://forum.openrefine.org/c/dev/8) and use the `election` tag. In your message, indicate

> * The name of the individual nominated with its GitHub or forum username

Author: ```suggestion
* The name of the individual nominated with their GitHub or forum username
```

> * For which group you are nominating this person (Committers, Core Developer or Code of Conduct Committee)

Author: ```suggestion
* Which group you are nominating this person (Committers, Core Developer or Code of Conduct Committee)
```

* Why are you nominating this person? You can list significant contributions and adherence to OpenRefine governance.

See, for example [this message](https://forum.openrefine.org/t/nominating-sunilnatraj-to-be-part-of-the-openrefine-github-organization/1914)

> Voting is open for 72 hours to provide an opportunity for all concerned persons to participate, regardless of their geographic location.

Author: ```suggestion
Voting is open for seven days to provide an opportunity for all concerned persons to participate, regardless of their geographic location.
```

Increasing to seven days as per @thadguidry  request. Based on how frequently a number of committers and core contributors interact, I think it makes sense to increase it so we do not create artificial pressure to vote.

> Voting is open for 72 hours to provide an opportunity for all concerned persons to participate, regardless of their geographic location.

Author: agree, it should be 7 days, I can't find the mention of 72h in the current version.

Only eligible voters have a binding vote. Votes are expressed as
- +1: yes
- 0: I will not oppose the nomination.

> - -1: no, I oppose the nomination. All -1 votes must be justified.

Author: ```suggestion
- -1: no, I oppose the nomination. All -1 votes must be explained.
```
